app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,73 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>ClassroomDashboard</title>
+    <title>Classroom Dashboard</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= csrf_meta_tags %>
 
+    <%= stylesheet_link_tag "application.css" %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= favicon_link_tag asset_path('favicon.ico'), :rel => 'icon', :type =>  'image/x-icon' %>
+
+  <% if logged_in? %>
+
+  <nav class="navbar navbar-default">
+  <div class="navbar-container">
+    <div class="navbar-header">
+      <button class="navbar-toggle collapsed" data-target="#bs-navbar-collapse-1" data-toggle="collapse" type="button">
+        <span class="sr-only">Toggle Navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <div class="navbar-brand">
+      <%= link_to "Classroom Dashboard", root_path %>
+      </div>
+    </div>
+
+    <div class="collapse navbar-collapse" id="bs-navbar-collapse-1" style="background-color:rgb(78,55,100);">
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+        <%= link_to "Courses", user_courses_path(current_user["id"]) %>
+        </li>
+        <li>
+        <%= link_to "Profile", user_path(current_user["id"]) %>
+        </li>
+        <li>
+        <%= link_to "Log Out", "/logout", method: :delete %>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+
+<% else %>
+<nav class="navbar navbar-default">
```

Comment:
The indentation seems off here. I think everything within this else statement should be indented more to match the if statement.